### ISSUE TITLE
IPlayTV AppleTV look and feel on the iPad.

### ISSUE THREAD

**Issue by MNLFDF** [andrefmsilva/iplaytvaio-bugs#106]
It make sense how the UI is for the iPhone app, but the iPad seems more a “big screen” to me, so why not make it the same look-and-feel as on the AppleTV. (Maybe forcing people to use the app always in landscape mode, although I don’t like myself to be forced how I should use my own device). 

I am not a developer, so I don’t know if this is easy to do, but would make the UI more future proof. Also don’t know if you make it so when the user holds its iPad in portrait mode, it get the old UI and when holding it in landscape mode you get the more “fancy” AppleTV look-and-feel.

iPlayTV users, please comment on this idea if you like it or not?

**Comment by Huxly72** [comment]
+1

**Comment by Austrapede** [comment]
I totally agree. A lot of space is wasted and the margins are not well adjusted in landscape mode.

Current design
![Captura 2022-11-10 a les 0 33 31](https://user-images.githubusercontent.com/64137086/200964813-5f27ef84-12f3-49da-9f36-65b4c0b1d223.png)

This could be a design proposal
![Captura 2022-11-10 a les 0 41 09](https://user-images.githubusercontent.com/64137086/200965590-0d3e6647-6106-4131-853d-0113763c3d51.png)

![Captura 2022-11-10 a les 0 42 04](https://user-images.githubusercontent.com/64137086/200965627-45e6631a-f189-484d-b5b1-7dbc2c9570e8.png)

**Comment by Lobello04** [comment]
+1
